Eevee.Sleep.Bot/Models/InGameAnnouncement/InGame/InGameAnnouncementDetailResponse.cs

```diff
@@ -0,0 +1,44 @@
+using System.Text.Json.Serialization;
+using Eevee.Sleep.Bot.Enums;
+using Eevee.Sleep.Bot.Extensions;
+
+namespace Eevee.Sleep.Bot.Models.InGameAnnouncement.InGame;
+
+public record InGameAnnouncementDetailResponse : IInGameAnnouncementResponse<InGameAnnouncementDetailModel> {
+    [JsonPropertyName("id")]
+    public required string Id { get; init; }
+
+    [JsonPropertyName("title")]
+    public required string Title { get; init; }
+
+    [JsonPropertyName("category")]
+    public required int Category { get; init; }
+
+    [JsonPropertyName("imgUrl")]
+    public required string ImgUrl { get; init; }
```

Comment:
Should be fine having full name here `ImageUrl`

Reply:
I think these API responses should match the actual field names returned by the API to avoid confusion.
The actual values are immediately converted to a Model, and the clean values should be handled there

Reply:
But `JsonPropertyName` has sort of the name mapping of it already, for example:

```
[JsonPropertyName("imgUrl")]
public required string ImageUrl { get; init; }
```

By this we can know that the source field is `imgUrl`, and just use maybe `model.ImageUrl` everywhere in the code to know that it's the image URL from the API.